api/src/main/java/org/confeti/controllers/core/ConferenceController.java

```diff
@@ -0,0 +1,75 @@
+package org.confeti.controllers.core;
+
+import lombok.RequiredArgsConstructor;
+import org.confeti.controllers.dto.core.ConferenceStatResponse;
+import org.confeti.service.ConferenceService;
+import org.confeti.service.ReportStatsService;
+import org.confeti.service.dto.stats.ReportStatsByConference;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import java.util.Map;
+
+import static org.confeti.controllers.ControllersUtils.CONFERENCE_NAME_URI_PARAMETER;
+import static org.confeti.controllers.ControllersUtils.REST_API_PATH;
+import static org.confeti.controllers.ControllersUtils.YEAR_URI_PARAMETER;
+import static org.confeti.controllers.core.StatisticControllerUtils.handleBaseGetRequest;
+import static org.confeti.controllers.core.StatisticControllerUtils.handleForAllRequest;
+import static org.confeti.controllers.core.StatisticControllerUtils.handleSpecifiedRequest;
+import static org.confeti.controllers.core.StatisticControllerUtils.handleSpecifiedRequestWithKey;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping(value = REST_API_PATH + "/conference", produces = APPLICATION_JSON_VALUE)
+@RequiredArgsConstructor
+public class ConferenceController {
+    private final ConferenceService conferenceService;
+    private final ReportStatsService reportStatsService;
+
+    @GetMapping
+    @ResponseBody
+    public Mono<ResponseEntity<?>> handleConferenceRequest() {
+        return handleBaseGetRequest(conferenceService.findAll());
+    }
+
+    @GetMapping(path = "{" + CONFERENCE_NAME_URI_PARAMETER + "}/stat", params = {YEAR_URI_PARAMETER})
+    @ResponseBody
+    public Mono<ResponseEntity<?>> handleRequestCompanyYear(
+            @PathVariable(CONFERENCE_NAME_URI_PARAMETER) final String conferenceName,
+            @RequestParam(YEAR_URI_PARAMETER) final int year) {
+        return handleSpecifiedRequestWithKey(reportStatsService.countConferenceStatsForYear(conferenceName, year),
+                stat -> new ConferenceStatResponse()
+                        .setConferenceName(conferenceName)
+                        .setYears(Map.of(year, stat.getReportTotal())));
+    }
+
+    @GetMapping(path = "{" + CONFERENCE_NAME_URI_PARAMETER + "}/stat")
+    @ResponseBody
+    public Mono<ResponseEntity<?>> handleRequestCompany(
```

Comment:
I think you mean `handleRequestConference`. And it is better to use `stat` in the name of methods because you have methods that do not deal with statistics but are named for example like [this](https://github.com/confeti/confeti/pull/55/files#diff-c61fca7eff49915b27fa44ae833c9ec7fab3c67c4ddb8f5ca5b7701bd7c6f47eR37)